CONTRIBUTING.md

```diff
@@ -0,0 +1,187 @@
+| `html` | Output filename for the page. Should end in `.html` and be unique within the target. |
+
+
+## Translations
+
+The XRP Ledger Dev Portal is mostly written in English, so the English version is generally the most up-to-date and accurate version. However, to broaden the reach of the XRP Ledger software and community, this repository also contains translated versions of the documentation. We strongly encourage members of the community who understand other languages to contribute translations of the dev portal contents in their native languages.
+
+The `dactyl-config.yml` contains a "target" entry for each available language. (As of 2019-11-18, the available languages are English and Japanese.) This entry includes a dictionary of strings used in the template files. For example:
+
+```yaml
+-   name: en
+    lang: en
+    display_name: XRP Ledger Dev Portal
+    # These github_ fields are used by the template's "Edit on GitHub" link.
+    #  Override them with --vars to change which fork/branch to edit.
+    github_forkurl: https://github.com/ripple/xrpl-dev-portal
+    github_branch: master
+    strings:
+        blog: "Blog"
+        search: "Search site with Google..."
+        bc_home: "Home"
+        # ...
+```
+
+There is also a top-level `languages` listing that defines some properties for each supported language. The short code for each language should be short code according to [IETF BCP47](https://tools.ietf.org/html/bcp47). For example, "en" for English, "es" for Spanish, "ja" for Japanese, "zh-CN" for Simplified Chinese, "zh-TW" for Traditional Chinese (as used in Taiwan), and so on. The `display_name` field defines the language's name as written natively in that language. The `prefix` field defines a prefix to be used in hyperlinks to that language's version of the site. Example `languages` definition:
+
+```yaml
+languages:
+    -   code: en
+        display_name: English
+        prefix: "/"
+    -   code: ja
+        display_name: 日本語
+        prefix: "/ja/"
+```
+
+The same `dactyl-config.yml` file contains an entry for each content page in the XRP Ledger Dev Portal. If a page has been translated, there is a separate entry for each translation, linked to the "target" for that translation. If a page has not yet been translated, the English version is used across all targets. For each page, the HTML filename (`html` field) and navigation fields (`funnel`, `doc_type`, `supercategory`, `category`, and `subcategory`, if supplied) should be the same across all language versions of a page. The fields that are different for translated versions of the page should be (in all cases, only if the entry uses the field):
+
+- **`name`** - the title of the page. This is usually only provided on landing pages with no Markdown source file, or special pages using unusual template, such as dev tools. This field is typically omitted from Markdown files, because Dactyl derives the title from the header on the first line of the file.
```

Comment:
```suggestion
- **`name`** - the title of the page. This is usually only provided on landing pages with no Markdown source file, or special pages using an unusual template, such as the [Dev Tools](dev-tools.html) page. This field is typically omitted from Markdown files, because Dactyl derives the title from the header on the first line of the file.
```